UN-OCHA/unocha-site
# [UNO-744] Metatags

Refs: UNO-744

This PR enables and configures the metatags similarly to what was on GHO 2022.

**Image styles**

It notably adds 4 image styles for the facebook and twitter images. 

2 of them apply a scale and crop effect like it was set up on GHO 2022 which works fine for regular images and results in the exact size expected by facebook and twitter.

The 2 other styles only scale up to the expected dimensions (either width or height depending on the ratio). This is to preserve the aspect ratio of the leader portraits and events.

That said I don't know what facebook/twitter will do with those images and they may not respect the aspect ratio.

**Site name**

This PR also changes the site name from `UN-OCHA` to `OCHA` to match the D7's site name.

**Notes**

The `twitter:image:width` and `twitter:image:height` were removed from the metatag module because they are deprecated apparently. So it's the difference between the UNO metatags and the GHO 2022 ones.

### Tests

1. Checkout the branch, run composer install to update the metatag module, clear the cache and import the config
2. Visit different types of pages (regions, stories etc.) and check that the metatags seem correct (title, description, url, og:image, twitter:image etc.) by comparing with the metatags of https://2022.gho.unocha.org/intro/foreword-emergency-relief-coordinator/ for example.
